newIDE/app/public/index.html

```diff
@@ -61,5 +61,20 @@
         </p>
       </div>
     </noscript>
+    <script>
+      (function() {
+        // Apply background color for startup
+        var backgroundColors = {
+          Dark: '#252525',
+          Nord: '#2E3440',
+          'Solarized Dark': '#00212B'
+        };
+        console.log(localStorage.getItem("gd-preferences"))
+        var themeName = JSON.parse(localStorage.getItem("gd-preferences"))['themeName'];
+        if (themeName != 'GDevelop default') {
```

Comment:
Maybe you should also check if `backgroundColors[themeName]` is defined? I am not sure though, it would show an error that shouldn't crash the ide but alert the developper he forgot to add a background color here.